docs/changes.rst

```diff
@@ -2,8 +2,14 @@
 Change Log
 ==========
 
+9.0.1
+-----
+
+- Do not fetch auto-generated HTML file 
```

Comment:
```suggestion
- Do not fetch index.cnxml.html files on `get`
```
Maybe?

Reply:
no, because I think we want to make clear _why_ we're excluding this funny-named file: it's not editable, it's a generated file.